__tests__/app.test.js

```diff
@@ -159,3 +159,85 @@ describe("GET /api/reviews/:review_id/commnets", () => {
       });
   });
 });
+//
+describe("POST /api/reviews/:review_id/commnets", () => {
+  it("201: should accept object with username and body and respond with posted comment", () => {
+    const review_id = 1;
+    const newComment = {
+      username: "mallionaire",
+      body: "what an awesome comment",
+    };
+    return request(app)
+      .post(`/api/reviews/${review_id}/comments`)
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toHaveProperty("comment_id", 7);
```

Comment:
I think here I would've been more tempted to allow any comment_id but I would expect the votes to be 0